File: packages/react-native/temp/package/ReactAndroid/src/main/java/com/facebook/react/devsupport/IInspectorPackagerConnection.java
/*
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

package com.facebook.react.devsupport;

/* package */ interface IInspectorPackagerConnection {
  public void connect();

  public void closeQuietly();

  public void sendEventToAllConnections(String event);
}

File: packages/react-native/ReactAndroid/src/main/java/com/facebook/react/devsupport/IInspectorPackagerConnection.kt
/*
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

package com.facebook.react.devsupport

internal interface IInspectorPackagerConnection {
  fun connect()

  fun closeQuietly()

  fun sendEventToAllConnections(event: String?)
}
